package/README.md

```diff
@@ -65,6 +65,12 @@ Here is an overview of the release process. See the section below for detailed i
 
 ## Creating a release (for maintainers)
 
+0. In case something goes wrong, release can be canceled:
+
+    * tag must be removed (`git tag -d x.y.z; git push --delete origin x.y.z`)
+
+    * release must be deleted: github > releases > edit x.y.z > remove all files > delete button appears
+
```

Comment:
How about adding a step here to open an issue for tracking the activities, then closing it at the end?

Reply:
Also, let's make that issue the 3rd sticky.